plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java

```diff
@@ -621,6 +621,58 @@ public void testBigQuerySnapshotTable()
         }
     }
 
+    @Test
+    public void testWildcardTable()
+    {
+        String suffix = randomTableSuffix();
+        String firstTable = format("test_wildcard_%s_1", suffix);
+        String secondTable = format("test_wildcard_%s_2", suffix);
+        String wildcardTable = format("test_wildcard_%s_*", suffix);
+        try {
+            onBigQuery("CREATE TABLE test." + firstTable + " AS SELECT 1 AS value");
+            onBigQuery("CREATE TABLE test." + secondTable + " AS SELECT 2 AS value");
+
+            assertQuery("DESCRIBE test.\"" + wildcardTable + "\"", "VALUES ('value', 'bigint', '', '')");
+            assertQuery("SELECT * FROM test.\"" + wildcardTable + "\"", "VALUES (1), (2)");
+
+            assertThatThrownBy(() -> query("DROP TABLE test.\"" + wildcardTable + "\""))
+                    .hasStackTraceContaining("com.google.cloud.bigquery.BigQueryException: Illegal operation");
```

Comment:
what's the top-level exception being raised? add assertion on it's message

Reply:
add tests with INSERT, ADD COLUMN, RENAME

Reply:
Fixed to throw an exception explicitly and added some cases.